### DISCUSSION TITLE
Register page

### DISCUSSION THREAD

**Discussion by chandan4744** [hotspotbilling/phpnuxbill#284] [Q&A]
register page (localhost/phpbill/index.php?_route=register) not working plz.. solve the problems @ibnux

**Comment by ibnux** [comment]
It works

**Comment by chandan4744** [comment]
https://github.com/user-attachments/assets/75c8f24d-eff8-4b00-bcc9-5cc546873562

not working..............
plz solve

**Reply by chandan4744** [reply]
@ibnux
